aiosonic/__init__.py

```diff
@@ -99,6 +99,7 @@ def __init__(self):
         self.compressed = b""
         self.chunks_readed = False
         self.request_meta = {}
+        self.loop = loop
```

Comment:
```suggestion
        self._loop = loop
```

also in `__del__` method, `self.loop` -> `self._loop`